opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperTestDialog.tsx

```diff
@@ -58,14 +71,21 @@ const CsvMapperTestDialog: FunctionComponent<CsvMapperTestDialogProps> = ({
       .then((data) => {
         const resultTest = (data as CsvMapperTestDialogQuery$data)
           .csvMapperTest;
-        setResult(JSON.stringify(resultTest, null, '  '));
+        setResult({
+          value: JSON.stringify(resultTest, null, '  '),
+          nbEntities: resultTest.filter((obj: ResultCSVMapper) => !obj.relationship_type).length,
+          nbRelationships: resultTest.filter((obj: ResultCSVMapper) => !!obj.relationship_type).length,
```

Comment:
Why !! ?

Reply:
We need to display : 
- number of entity found
- number of relationship found
So I thought it's was better to provide the information here. 

How should I handle it ?

Reply:
Just asking about the double negation on the relationship_type.

Reply:
For me having a double negation would show my intention to return a boolean value. But I can remove it if you prefer it to be written without double negation.

Reply:
Ok, was just checking that I was not missing something about the obj type :) Good for me.